format-objc-file.sh

```diff
@@ -68,12 +75,17 @@ function format_objc_file_dry_run() {
 		return
 	fi
 
+	style="-style=file"
+	if [ ! -z $CONFIG_FILE ]; then
+		style="-style=$CONFIG_FILE"
```

Comment:
Does this need to be:
```suggestion
		style="-style=file:$CONFIG_FILE"
```